### PR TITLE
Main Page Charts

### PR DESCRIPTION
**Pull request by asedarski**

### CODE REVIEW

**Comment by asedarski** [comment]
Issue #9 
Issue #10

**Comment by asedarski** [comment]
Issue #59

**Comment by asedarski** [comment]
https://github.com/Code4HR/hercules/issues/59

**Comment by asedarski** [comment]
Issue #12 
Issue #13

File: public/js/main.js [hunk 1]
```diff
@@ -1,222 +1,193 @@
...
+                {
+                    data: [323533, 395871, 320429, 350098, 190527, 85852],
+                    backgroundColor: ['#6EC9AC', '#7715F7', '#316BBF', '#778899', '#085F61', '#0F9CE5'],
+                    hoverBackgroundColor: ['#d6dbe0', '#d6dbe0', '#d6dbe0', '#d6dbe0', '#d6dbe0', '#d6dbe0']
+                }
+            ]
+        };
+
+        new Chart($('#populationByAge'), {
+            type: 'doughnut',
```
Comment (hop-along-polly) [inline public/js/main.js:53]: Mmmhmmm doughnuts

File: public/js/main.js [hunk 2]
```diff
@@ -1,222 +1,193 @@
...
-                pointColor: 'rgba(38, 185, 154, 0.85)',
-                pointStrokeColor: '#fff',
-                pointHighlightFill: '#fff',
-                pointHighlightStroke: 'rgba(151,187,205,1)',
-                data: [28, 48, 40, 19, 96, 27, 100]
-            }
-        ]
-    };
-
+    // // Population by race
```
Comment (hop-along-polly) [inline public/js/main.js:58]: Double comment wha?!?!

File: public/js/main.js [hunk 3]
```diff
@@ -1,222 +1,193 @@
...
     $(document).ready(function () {
-        window.myPolarArea = new Chart(document.getElementById('canvas_area').getContext('2d')).PolarArea(sharePiePolorDoughnutData, {
-            responsive: true,
-            tooltipFillColor: 'rgba(51, 51, 51, 0.55)'
+        var ratingByEnrollment;
+
+        // Gather aggregates from Hercules
+        $.ajax({
+            method: 'GET',
+            url: 'http://hercules.code4hr.org/schools?groupBy=enrollment&averageField=gsRating',
```
Comment (hop-along-polly) [inline public/js/main.js:83]: ParentRating is on more records than gsRating...either is fine though

File: public/js/main.js [hunk 4]
```diff
@@ -1,222 +1,193 @@
...
+                        }
+                    ]
+                };
+
+                var options = {
+                    scales: {
+                        yAxes: [{
+                            scaleLabel: {
+                                display: true,
+                                labelString: 'Number of Enrollment'
```
Comment (hop-along-polly) [inline public/js/main.js]: Number of enrolled students is a better description

File: src/Explore/Views/main.twig [hunk 5]
```diff
@@ -48,176 +54,198 @@
...
+                    <li class='presentation'><a href='#schools'><i class='glyphicon glyphicon-apple'></i>  Schools</a></li>
+                    <li class='presentation'><a href='#people'><i class='glyphicon glyphicon-user'></i>  People</a></li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <script src='/js/main.js'></script>
+
+</body>
```
Comment (hop-along-polly) [inline src/Explore/Views/main.twig]: Can you add. Newline at the end of the file kthxbai

**Comment by hop-along-polly** [comment]
Killin it @asedarski keep being awesome! :+1:

File: public/js/main.js (hunk 3 above)
Comment (asedarski) [reply public/js/main.js:83]: I chose gsRating because that one is 1-10 instead of 1-5... gave better graphics :)

**Comment by hop-along-polly** [comment]
Who am I to interfere with smexy graphics

File: public/js/main.js (hunk 4 above)
Comment (asedarski) [reply public/js/main.js]: Number of Enrolled Students doesn't fit on the axis.. How about Students Enrolled?

Comment (asedarski) [reply public/js/main.js]: Or # Students Enrolled

**Comment by hop-along-polly** [comment]
How about yes

**Comment by asedarski** [comment]
@ezzy1337 Updated! Also added an update to the readme

**Pull request merged**